net-misc/openssh/openssh-9.8_p1-r1.ebuild

```diff
@@ -138,6 +138,12 @@ src_prepare() {
 		-e 's:-D_FORTIFY_SOURCE=2::'
 	)
 
+	if use elibc_musl; then
+		sed_args+=(
+			-e '/SYSTEMD_NOTIFY/d'
+		)
+	fi
+
```

Comment:
I greatly prefer the original patch to this crappy hackaround.

Reply:
lol. I guess that makes one more "crappy hackaround" in this ebuild then. It's whatever you all want. It's all already fixed by not using this hackaround on my system.

Reply:
Sorry, I didn't mean to call your work crappy, especially after someone else asked you to do it.

I simply disagree with this as a "solution".